README.md

```diff
@@ -30,7 +30,6 @@ Here's an example about some basic features, it might also help to read through
 
 ````python
 from ukis_pysat.data import Source
-from ukis_pysat.raster import Image
```

Comment:
this is still needed here, see line 65

Reply:
Sorry didnt see that one. Thanks for correcting.